src/Components/footer/FooterStyles.js

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+
```

Comment:
Please use plain CSS in our FYP project.
Since you have putted effort, its appreciated. It wont take long to convert it into simple CSS. You can use google or contact me in case of any hurdles.